TESTING.rst

```diff
@@ -621,6 +726,53 @@ print output generated test logs and print statements to the terminal immediatel
 You can modify the tests or KubernetesPodOperator and re-run them without re-deploying
 airflow to KinD cluster.
 
+
+Sometimes there are side effects from running tests. You can run ``redeploy_airflow.sh`` without
+recreating the whole cluster. This will delete the whole namespace, including the database data
+and start a new Airflow in the cluster.
```

Comment:
```suggestion
recreating the whole cluster. This will delete the whole namespace, including the database data
and start a new Airflow deployment in the cluster.
```